### ISSUE TITLE
Prolinkování rešerše o energetických scénářích

### ISSUE THREAD

**Issue by ondraspribyla** [faktaoklimatu/web-cz#804]
Bylo by podle mě super, abychom mohli prolinkovat rešerši o energetických scénářích s jednotlivými studiemi. (např s nyní doplňovanou studií IDEA... nemyslím tím ale odkaz z textu (to podle mě máme), ale odkaz s náhledem na pravém boku stránky - což asi znamená udělat nějaký grafický zpracování "rešerše" ?? @jankrcal @mgrabovsky

**Comment by jankrcal** [comment]
Rešerše už mají náhled, ale je trochu nudný :)
<img width="364" alt="Screen Shot 2021-03-24 at 15 37 27" src="https://user-images.githubusercontent.com/145227/112328667-d925c600-8cb6-11eb-8ec5-39799b49f47d.png">

K tomu jsem dřív volně přemýšlel:

Rešerše jsou náš produkt (např. narozdíl od studií).
Současně rešerše nemají jasný vizuální výsledek (narozdíl od infografik).
Co takhle zpracovat rešerše podobně jako explainery, tzn. jim dát nějaký tématický velký cover obrázek?

Co myslíš, Ondráši?

**Comment by ondraspribyla** [comment]
Souhlasím, myslím, že textově stačí rešerši napsat krátký úvod, který vymezuje téma rešerše... Cover obrázek by pak dával smysl...
